README.md

```diff
@@ -5,7 +5,57 @@ This code is subject to the terms of the BSD+Patent license.
 See LICENSE file for more details.
 -->
 
-## HOWTO build
+# RDKB (OE 2.2)
+
+## Raspberry Pi 3 B+
+
+### HOWTO build
+
+> Based on [official RDK build manual](https://wiki.rdkcentral.com/pages/viewpage.action?pageId=71011616#RDK-B(RaspberryPi3B+)BuildandSetupManual-RouterProfile-BuildInstructions)
+
+* Fetch sources
+```
+repo init -u https://code.rdkcentral.com/r/manifests -b rdk-next -m rdkb-extsrc.xml
+repo sync -j4 --no-clone-bundle
+```
+
+* Fetch meta-prplmesh
+```
+git clone -b rdkb-next https://github.com/prplfoundation/meta-prplmesh
+```
+
+* Source build env
```

Comment:
Maybe better to create configuration script which will add all necessary changes into the build config (bblayers.conf, etc) ?